transport_nantes/press/views.py

```diff
@@ -128,7 +128,16 @@ def fetch_opengraph_data(request, url=None, is_view=False) \
         url = request.POST["url"]
 
     try:
-        response = requests.get(url)
+        # Some websites may filter the default user agent python-requests
+        # We use a custom one to avoid this
+        headers = {
+            # Found this user agent on https://stackoverflow.com/a/57155793/16279937
+            "User-Agent": ("Mozilla/5.0 (Windows NT 10.0; Win64; x64)"
```

Comment:
Avoid magic values.  Put these in variables, with explanations of why you are using them.  Not just that you found them on SO, but what further checks you did to validate that the SO values were good.  For example, this question is almost four years old.  When would a browser with this UA string have been released?

Is it realistic that a UA string would claim to be mozilla on Win NT 10, AppleSafari, and Chrome all at the same time?  Again, this needs documentation to show what you learned in doing due diligence and that this isn't just blind "copy, paste, it works, done" without further thought.

Reply:
It's not an unbelievable UA, I've replaced the snippet with my today's user agent : 
![image](https://user-images.githubusercontent.com/70256364/205594091-78824b62-fc00-4112-9a04-28b6b3077724.png)

So you have full breakdown of it.
 Win NT 10 -- OS
AppleSafari -- The Web Kit provides a set of core classes to display web content in windows
Chrome -- Actual browser

The UA had a quite old version of chrome, so I took this bit of research as an opportunity to use a more recent one. The goal here is to not be filtered by robots.txt